### ISSUE TITLE
How long takes to train ASFF model?

### ISSUE THREAD

**Issue by y0un5** [ruinmessi/ASFF#24]
In the paper, you said that trained ASFF model for 300 epochs on 4 V100 GPUs. Could you share how long time you spent for 300 epochs training?

> The entire network is trained with stochastic gradient descent (SGD) on 4 GPUs (NVDIA Tesla V100) with 16 images per GPU. All models are trained for 300 epochs with the first 4 epochs of warmup and the cosine learning rate schedule [26] from 0.001 to 0.00001.

**Comment by ruinmessi** [comment]
about 6 days in 4 V100, one week in 10 2080ti.

**Comment by y0un5** [comment]
Thanks!
